typescript.js

```diff
@@ -1,5 +1,3 @@
-'use strict';
-
 module.exports = {
-  extends: ['./base.js', 'airbnb-typescript/base', 'plugin:@typescript-eslint/recommended'],
+  extends: ['./back.js', 'airbnb-typescript/base', 'plugin:@typescript-eslint/recommended'],
```

Comment:
the base.js is no longer exported? 🤔

Reply:
It is. 

You can use it by extending `strapi`. The back config is available by doing `strapi/back`.